docs/modules/ROOT/pages/basic-concepts.adoc

```diff
@@ -3,7 +3,7 @@
 
 In this section we'll explain the basic concepts of Kafka Connect and Camel and how we can create a relation between the two frameworks
 
-As reported on the Apache Kafka website: _Kafka Connect is a tool for scalably and reliably streaming data between Apache Kafka and other systems. It makes it simple to quickly define connectors that move large collections of data into and out of Kafka. Kafka Connect can ingest entire databases or collect metrics from all your application servers into Kafka topics, making the data available for stream processing with low latency. An export job can deliver data from Kafka topics into secondary storage and query systems or into batch systems for offline analysis._
+As reported on the Apache Kafka website: _Kafka Connect is a tool for scalable and reliably streaming data between Apache Kafka and other systems. It makes it simple to quickly define connectors that move large collections of data into and out of Kafka. Kafka Connect can ingest entire databases or collect metrics from all your application servers into Kafka topics, making the data available for stream processing with low latency. An export job can deliver data from Kafka topics into secondary storage and query systems or into batch systems for offline analysis._
```

Comment:
In this case, I think it should be rephrased as "scalable and reliably data streaming between" ...

Reply:
Yes